### DISCUSSION TITLE
Libre2 US/CA/AUS Compatibility

### DISCUSSION THREAD

**Discussion by maerlyn13** [NightscoutFoundation/xDrip#1937] [Q&A]
Is there any way to implement us/AU/ca libre2 support into xdrip? Diabox has supported these sensors directly through Bluetooth for awhile and the developers of Diabox stated they would share the code with anyone who asks for it.

**Comment by Navid200** [comment]
@tolot27 Do we need a "device-libre2" label?  Or, should I label them all with device-libre?

**Reply by tolot27** [reply]
I think it is sufficient to stick with "device-libre". There are not too many issues/discussions labeled with it.

**Reply by jaysondennis** [reply]
Hi All.... does this mean this suggestion/ask is being worked on?  Is there any work being done to connect xdrip to the freestyle libre 2 Canada sensors? The rest of world has been connected for some time but seems canada has been left behind here. There is anoth solution, diabox but xdrip is far superior in my opinion and would allow me to ise my watch as collector. Let me know. Thanks!

**Reply by maerlyn13** [reply]
I don’t really think the devs care about compatibility with libre2 AU/US/CA. They will tell you to just use Diabox and avoid the question completely. There must be some animosity between xdrip devs and diabox.

**Reply by jaysondennis** [reply]
I mean diabox works but xdrip is by far better.  Diabox losses connection all the time and I need internet to run all the other tools diabetics need like nightscout and androidaps... Hopefully the dev that sent me to this discussion can make something happen.  Thanks

**Reply by Navid200** [reply]
The developers are volunteers.  There is no way to tell a developer what to do, or not to do.  There is no incentive for a developer to do anything other than love.  If the developer doesn't feel an urge to fix something, there is nothing I, or anyone else, can do to encourage them to.

I have suggested it.  There are challenges.  It's not an easy thing to do.  And none of the developers themselves who use Libre live in North America.  
There is a similar discussion on facebook.  I have said the same thing over there.  There is a guy who showed interest.  He has already made a successful edit to xDrip.  
But, I have not heard anything from him.

**Reply by jaysondennis** [reply]
Thanks Navid200... First off we all love your collective work and for sure realize that you are volunteering out of love.  Nothing other than that crossed my mind... I am just a very interested party who cannot develop but knows enough to talk like someone who almost could :).... If you hear from that Facebook person and they have a fork of xdrip I can try let me know :) thanks again!!!

**Reply by jaysondennis** [reply]
No one found a link to that fork for Canada libre 2 in xdrip?

**Comment by Navid200** [comment]
@tzachi-dar

**Reply by jaysondennis** [reply]
@tzachi-dar any thoughts? Thanks

**Comment by tzachi-dar** [comment]
Sorry, but I will not have time for it in the foreseen future.
To be honest I must say that I did not get enough information from the diabox developers.
